isic/core/search.py

```diff
@@ -100,6 +106,8 @@ def bulk_add_to_search_index(qs: QuerySet[Image], chunk_size: int = 2000) -> Non
         if not success:
             logger.error('Failed to insert document into elasticsearch', info)
 
+    opensearch_logger.setLevel(previous_log_level)
```

Comment:
@danlamanna If the code above raises an exception, this will not be re-sent. A context manager will make this more robust. It should be easy to make one with `contextlib`.